test/lib/linter/rules/snapshots/BestPractices.ts.md

```diff
@@ -0,0 +1,271 @@
+# Snapshot report for `test/lib/linter/rules/BestPractices.ts`
+
+The actual snapshot is saved in `BestPractices.ts.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## General: Negative_1/Component.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'Negative_1/Component.js',
+        messages: [],
+        warningCount: 0,
+      },
+    ]
+
+## General: Negative_2/Component.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'Negative_2/Component.js',
+        messages: [],
+        warningCount: 0,
+      },
+    ]
+
+## General: Negative_3/Component.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'Negative_3/Component.js',
+        messages: [],
+        warningCount: 0,
+      },
+    ]
+
+## General: Negative_4/Component.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'Negative_4/Component.js',
+        messages: [],
+        warningCount: 0,
+      },
+    ]
+
+## General: Negative_5/Component.js
+
+> Snapshot 1
+
+    [
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: 'Negative_5/Component.js',
```

Comment:
Why are there two files with the same path listed in the results?

Reply:
Fixed it. It was because the tests were using testnames for that